src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -441,38 +439,109 @@
+                         :disabledFromChainIDs disabled-from-chain-ids
+                         :disabledToChainIDs   disabled-to-chain-ids
+                         :gasFeeMode           gas-rates
+                         :fromLockedAmount     from-locked-amount}
+                        rest-keys)]]
      {:db            (cond-> db
                        :always (assoc-in [:wallet :ui :send :amount] amount)
                        :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
                        :always (assoc-in [:wallet :ui :send :sender-network-values]
                                 sender-network-values)
                        :always (assoc-in [:wallet :ui :send :receiver-network-values]
                                 receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
                        :always (update-in [:wallet :ui :send] dissoc :network-links)
                        token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+      :json-rpc/call [{:method   "wallet_getSuggestedRoutesV2Async"
+                       :params   params
+                       :on-error (fn [error]
+                                   (rf/dispatch [:wallet/suggested-routes-error error])
+                                   (log/error "failed to get suggested routes (async)"
+                                              {:event  :wallet/get-suggested-routes
+                                               :error  error
+                                               :params params}))}]})))
+
+(rf/reg-event-fx :wallet/stop-get-suggested-routes
+ (fn [{:keys [db]}]
+   {:db            (assoc-in db [:wallet :ui :send :loading-suggested-routes?] true)
+    :json-rpc/call [{:method   "wallet_stopSuggestedRoutesV2AsyncCalcualtion"
+                     :params   []
+                     :on-error (fn [error]
+                                 (log/error "failed to get suggested routes"
+                                            {:event :wallet/stop-get-suggested-routes
+                                             :error error}))}]}))
+
+(defn- transform-new-to-old-path
```

Comment:
Maps the v2 paths to the (old) router paths.

Destop's implementation https://github.com/status-im/status-desktop/pull/15344/files#diff-1da937d55df9cf5890ba3e3081a0a5dc254fefdf76b5b2f7f8b408183eaac940R12-R74